src/suffer.cpp

```diff
@@ -1398,10 +1398,10 @@ void Character::suffer_from_exertion()
     }
 
     // Significantly slow the rate of messaging when in an activity
-    int chance = activity ? 2000 : 60;
+    int chance = activity ? 2000 : 1;
     if( attempted_activity_level > max_activity && one_in( chance ) && !in_sleep_state() ) {
         add_msg_if_player( m_bad,
-                           _( "You're tiring out, continuing to work at this rate will be slower." ) );
+                           _( "You're tiring out, continuing to work at this rate will be slower. %s / %s" ), attempted_activity_level, max_activity );
```

Comment:
3. This is really spammy, I don't think there's much reason for it to be so much more often.
4. The display at the end is the floating point numbers that these activities correspond to, which means nothing. These should be either names, or just be removed. And I'm not sure this is of much value to display in addition to the panel.

Reply:
This was a git mistake from my trying to figure out a way to get the exertion bar to load.  Didn't realize I had committed it.